exchanges/deribit/deribit_wrapper.go

```diff
@@ -780,10 +787,21 @@ func (d *Deribit) GetOrderInfo(ctx context.Context, orderID string, _ currency.P
 			return nil, fmt.Errorf("%v: orderStatus %s not supported", d.Name, orderInfo.OrderState)
 		}
 	}
+	var tif order.TimeInForce
+	switch orderInfo.TimeInForce {
+	case "good_til_cancelled":
+		tif = order.GoodTillCancel
+	case "good_til_day":
+		tif = order.GoodTillDay
+	case "fill_or_kill":
+		tif = order.FillOrKill
+	case "immediate_or_cancel":
+		tif = order.ImmediateOrCancel
+	}
```

Comment:
You can attach PostOnly 
```go
if orderInfo.PostOnly {
tif |= order.PostOnly
}
```

And use the above switch case in a func and apply to all TODO: statements below.